### ISSUE TITLE
Is it possible to place multple windows in a single profile?

### ISSUE THREAD

**Issue by unclemusclez** [Hotrian/OpenVRDesktopDisplayPortal#59]
I would like stack two separate chat windows when i look behind my controller while live streaming. Is this possible?

**Comment by Hotrian** [comment]
You could run OVRDDP twice, and configure the two overlays to be side by side or stacked, and have each program targeting a different window to capture. Is this what you mean?

Edit: Sorry, my mobile app didn't tell me the title of the issue before I replied. You can't currently have two windows in a single profile, but you can save a profile for each window. This is how I did it when I was playing American Truck Simulator - one profile per overlay, running OVRDDP 3 times, I have an overlay on my dash, an overlay above me on the sun visor, and another overlay on my passenger seat. 

My local source can easily capture my full monitor (2560x1440) at 150fps up to around 500fps if things are changing fast enough - the API I'm working on only updates when something on the screen changed so it uses a lot less CPU as it only takes a 'screenshot' when necessary, so it isn't an issue to capture a bunch of windows, but I can't speak towards the performance of the public release with multi-window capture. 

I'll work on adding multi-window/multi-overlay functionality, but for now you can just run OVRDDP multiple times.

**Comment by Hotrian** [comment]
To add onto my Edit, I took the source closed source and plan on releasing on Steam. I went over my reasoning [in this post on Reddit](https://www.reddit.com/r/Vive/comments/5d4576/so_i_rigged_up_a_solution_for_keeping_an_eye_on/da2au3v/?context=10000). I plan to leave the current source available, but I'm planning on posting the new version to Steam soon. I've been in talks with Valve so whenever they get moving forward on that end it should be available on Steam. If they don't let me post it on Steam for whatever reason, I'll be releasing my local source as builds here on GitHub, but I won't be updating the public source code for the foreseeable future. 

Edit: Fixed the link.
